es/chain/epoch.js

```diff
@@ -26,7 +25,7 @@ async function sendTransaction (tx, options = {}) {
 }
 
 async function balance (address, { height, hash } = {}) {
-  return parseBigNumber((await this.api.getAccountByPubkey(address, { height, hash })).balance)
+  return (await this.api.getAccountByPubkey(address, { height, hash })).balance.toString()
```

Comment:
Is `toString()` necessary if you are using `json-bigint` with `storeAsString` option?

Reply:
`json-bigint` convert to string only bignumber value. If balance not a `bignumber` it will not parse it.